challenge_3/python/popcorn_anachronism/src/challenge_3.py

```diff
@@ -0,0 +1,16 @@
+a = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7]
+
+def majority_element(array):
+    n = len(array)
+    array_freq = {}     #dict of frequencies of items in array
```

Comment:
I would use the [Counter](https://docs.python.org/dev/library/collections.html#collections.Counter) class here. Keep in mind that using `item in array` means that you're reading through that array every time. This is trivial for small lists, but for large ones, you're implementation may approach growing at O(n^2) instead of just O(n)